src/components/clusterWizard/ClusterWizardForm.tsx

```diff
@@ -8,67 +9,73 @@ import {
   ButtonVariant,
   Grid,
   GridItem,
+  Alert,
+  AlertVariant,
+  AlertActionCloseButton,
+  TextInputTypes,
 } from '@patternfly/react-core';
-import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 
 import ClusterWizardToolbar from './ClusterWizardToolbar';
 import PageSection from '../ui/PageSection';
 import { ToolbarButton, ToolbarText } from '../ui/Toolbar';
-import { InputField, TextAreaField } from '../ui/formik';
+import { InputField, TextAreaField, SelectField } from '../ui/formik';
 import validationSchema from './validationSchema';
-import { ClusterDefinition } from '../../types/clusterDefinition';
-import { postInstallConfig } from '../../api/clusterDefinition';
 import GridGap from '../ui/GridGap';
-import { Cluster } from '../../api/types';
-import { Link } from 'react-router-dom';
+import { Cluster, ClusterUpdateParams, Host } from '../../api/types';
 import { WizardStep } from '../../types/wizard';
+import { patchCluster } from '../../api/clusters';
+import AlertsSection from '../ui/AlertsSection';
 
 interface ClusterWizardFormProps {
   cluster: Cluster;
   setStep: React.Dispatch<React.SetStateAction<WizardStep>>;
 }
 
 const ClusterWizardForm: React.FC<ClusterWizardFormProps> = ({ cluster, setStep }) => {
-  const initialValues: ClusterDefinition = {
-    clusterName: cluster.name || '',
-    DNSDomain: '',
-    openshiftVersion: '',
-    apiVIP: '',
-    dnsVIP: '',
-    ingressVIP: '',
-    machineCIDR: '',
-    pullSecret: '',
-    sshPrivateKey: '',
-    sshPublicKey: '',
+  const initialValues: ClusterUpdateParams = {
+    name: cluster.name || '',
+    openshiftVersion: cluster.openshiftVersion || '4.4',
+    baseDnsDomain: cluster.baseDnsDomain || '',
+    clusterNetworkCIDR: cluster.clusterNetworkCIDR || '',
+    clusterNetworkHostPrefix: cluster.clusterNetworkHostPrefix || 0,
+    serviceNetworkCIDR: cluster.serviceNetworkCIDR || '',
+    apiVip: cluster.apiVip || '',
+    dnsVip: cluster.dnsVip || '',
+    ingressVip: cluster.ingressVip || '',
+    pullSecret: cluster.pullSecret || '',
+    sshPublicKey: cluster.sshPublicKey || '',
+    hostsRoles:
+      cluster.hosts?.map((host: Host) => ({
+        id: host.id,
+        role: host.role as 'master' | 'worker',
```

Comment:
Worth of it's own type. List of these roles is expected to be changed in the future.

Reply:
I'll remove the hostRoles definition here for now as it is not needed.